<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>Differential-expression-analysis-and-replication by Arhodes-CGRB-OSU</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">Differential-expression-analysis-and-replication</h1>
      <h2 class="project-tagline">Differential Expression Analysis and Replication</h2>
      <a href="https://github.com/Arhodes-CGRB-OSU/Differential-Expression-Analysis-and-Replication/blob/gh-pages/README.md" class="btn">Start Here</a>
    </section>

    <section class="main-content">
      <h3>
<a id="statistical-analysis-for-differential-expression-data" class="anchor" href="#statistical-analysis-for-differential-expression-data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Statistical Analysis for Differential Expression Data</h3>

<p>Good biological inference from Next Generation Sequencing Data requires a basic understanding of statistical analysis developed specifically for data sets that contain millions of sequences across a small number of samples.</p>

<p>For RNA-seq data, the strategy taken is to count the number of reads that fall into annotated genes and to perform statistical analysis on the table of counts to discover quantitative changes in expression levels between experimental groups. Easy, right? Not, exactly.</p>

<ul>
<li><p>We have integer counts and not continous measurements. Data is not normally distributed, so statistical methods we applied to microarray data don't work here.</p></li>
<li><p>Replication levels in designed RNA-Seq experiments tend to be modest, often not much more than two or three. As a result, there is a need for statistical methods that perform well in small-sample situations.</p></li>
<li><p>There is a dependence of variance on the mean (which changes with increasing number of replicates)</p></li>
</ul>

<p><code>Solution</code>: Appropriate modelling of the mean-variance relationship in DGE data is important for making inferences about differential expression. Employing methods which assess the mean-variance relationship to help with the problem of estimating biological variability for experiments with small number of replicates.</p>

<p>In this module, learners will use R Statistical Software to walk-through activities designed to compare the performance of different tools (edgeR, DESeq2, limma-voom) for differential expression analysis of RNA-Seq data, and how the mean-variance relationship is addressed in datasets with increasing number of replicates.</p>

<h3>
<a id="learning-objectives" class="anchor" href="#learning-objectives" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Learning Objectives</h3>

<ul>
<li>understanding the relationship between mean and variance and how that changes with the number of replicates</li>
<li>familariizing with tools for DE analysis in R</li>
<li>understanding the importance of having replicates in a RNA-Seq study</li>
</ul>

<h3>
<a id="authors-and-contributors" class="anchor" href="#authors-and-contributors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Authors and Contributors</h3>

<p>This tutorial is based on a presentation by Meeta Mistry from the Harvard School of Public Health (<a href="https://github.com/mistrm82" class="user-mention">@mistrm82</a>), adapted for use in the Winter 2016 RNA-Seq Analysis Module II Workshop on Differential Expression presented by Adelaide Rhodes (<a href="https://github.com/Arhodes-CGRB-OSU" class="user-mention">@Arhodes-CGRB-OSU</a>), Center for Genome Research and Biocomputing at Oregon State University.</p>

      <footer class="site-footer">
        <span class="site-footer-owner"><a href="https://github.com/Arhodes-CGRB-OSU/Differential-Expression-Analysis-and-Replication">Differential-expression-analysis-and-replication</a> is maintained by <a href="https://github.com/Arhodes-CGRB-OSU">Arhodes-CGRB-OSU</a>.</span>

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>
